imaging/ml/ml_codelab/breast_density_auto_ml.ipynb

```diff
@@ -1,935 +1,961 @@
 {
-  "cells": [
```

Comment:
Can you please fix the indentation? Otherwise it's very hard to review.

Reply:
it seems already fixed in target branch by https://github.com/GoogleCloudPlatform/healthcare/pull/431